### PR TITLE
[homework #11] - js-advanced-3 by David Azuaje

### PR DESCRIPTION
**Pull request by dazuaje**
Hi Mike!

If there is anything that I should fix, please let me know.

### CODE REVIEW

File: homeworks/david-azuaje/11-js-advanced-03/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    throw new RangeError(`Message contains ${msg.length} characters!`);
+
+  if (msg.match(/(< ?[a-z]+ ?>)/i)) return false;
+
+  return true;
+};
+console.log(validateMessage("<img>"));
+
+
+// Task 2) => The API It doesn't working
```
Comment (solvd-mmelnyk) [inline homeworks/david-azuaje/11-js-advanced-03/index.js:21]: what do you mean on "The API It doesn't working"?

File: homeworks/david-azuaje/11-js-advanced-03/task-5/fetch.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+//Fetch
+
+const root = document.querySelector(".root");
+const fetchData = async () => {
+  try {
+    let url = "https://randomuser.me/api/?gender=female&results=10";
```
Comment (solvd-mmelnyk) [inline homeworks/david-azuaje/11-js-advanced-03/task-5/fetch.js:6]: please use const instead of let if variable does not changed

**Review by solvd-mmelnyk — COMMENTED** [review]
@dazuaje 
In general looks good but please fix next things:
1. Please format code with prettier
2. Please use `const` and `let` correctly

let me know if you have any questions!

File: homeworks/david-azuaje/11-js-advanced-03/index.js (hunk 1 above)
Comment (dazuaje) [reply homeworks/david-azuaje/11-js-advanced-03/index.js]: Task 2 Done

File: homeworks/david-azuaje/11-js-advanced-03/task-5/fetch.js (hunk 2 above)
Comment (dazuaje) [reply homeworks/david-azuaje/11-js-advanced-03/task-5/fetch.js]: Fixed

**Review by solvd-mmelnyk — APPROVED** [review]

**Pull request merged**